nullaway/src/test/java/com/uber/nullaway/NullAwayJSpecifyGenericsTests.java

```diff
@@ -662,6 +658,55 @@ public void ternaryMismatchedAssignmentContext() {
         .doTest();
   }
 
+  @Test
+  public void parameterPassing() {
+    makeHelper()
+        .addSourceLines(
+            "Test.java",
+            "package com.uber;",
+            "import org.jspecify.annotations.Nullable;",
+            "class Test {",
+            "static class A<T extends @Nullable Object> { }",
+            "  static A<String> sampleMethod(A<A<String>> a1, A<String> a2) {",
+            "     return a2;",
+            "  }",
+            "  static void testPositive(A<A<@Nullable String>> a1, A<String> a2) {",
+            "    // BUG: Diagnostic contains: Cannot pass parameter of type",
+            "    A<String> a = sampleMethod(a1, a2);",
```

Comment:
How about the other way around? This is invariant, right? So we should also test that we can't pass `A<String>` to something expecting `A<@Nullable String>`, correct?